src/RF/e2bp.cpp

```diff
@@ -269,11 +270,9 @@ bool E2bp::pressAndHoldFor(unsigned long duration) {
 // Fill a given buffer with the correct payload and return a pointer to it
 uint8_t* E2bp::getPayload(uint8_t* buf, PayloadType type) {
     buf[0] = 0x00;
-    buf[1] = device->getMode() == SHUTTER ? 0x16 : 0x04;
+    buf[1] = device->getMode() == SHUTTER ? 0x06 : 0x04;  // 0x16 = Drive all shutters connected to the bus.
     buf[2] = 0x00;
-    buf[3] = 0x20;
-    buf[4] = device->getHardwareAddress()[0];
-    buf[5] = device->getHardwareAddress()[1];
+    buf[3] = 0x00;  // Initial project was using 0x20. Might be used to drive specific device over the bus, using another device as gateway?
```

Comment:
I will test this on Yokis lights to see if this is working as expected.

Reply:
Putting back my original comment:

My devices are connected together in a bus.
I figured out that if I send:
B9 06 00 00 00 00 ?? 00 00 (for example) one is moving
B9 16 00 00 00 00 ?? 00 00 (for example) all are moving
I always address the device I want to move because of the implementation of the commands.

Now from our discussion I'm thinking that maybe if I address another device (gateway), but put the address of the device I want to drive in the command, such as:
B9 06 00 00 ?? 00 00 sent to the gateway
or maybe:
B9 06 00 20 ?? 00 00 sent to the gateway
The other device may move? Gateway device propagate the command through the bus?
That could be interesting because it allows the mqtt gateway to be far away of some devices.

I didn't tested it. I saw this 0x20 on your firmware but I guess we have different setup.

I'm pretty sure it's possible to test it with other devices, using simple commands such as ON, and
B9 04 or B9 14 to address all.

If it's working as I expect it's a bigger change because the mqtt gateway could try to address any device it knows that is flagged as "part of bus" to reach transparently another one.